src/tir/schedule/primitive/block_annotate.cc

```diff
@@ -16,6 +16,9 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+#include <tvm/runtime/container/array.h>
```

Comment:
Do we need to include this explictly?

Reply:
Actually not. See utils.h

Reply:
Yeah you're right. But it's conventional to put `#include "../../ir/functor_common.h"` before `#include "../utils.h"`, in which case Array wasn't imported be fore `functor_common.h`.. I want to know that, is it acceptable to put `utils.h` before `functor_common.h`, or is there any other way 😥 ?

Reply:
You should add this include in functor_common.h

Reply:
Ah that makes sense, thanks!